repositories.props

```diff
@@ -3,17 +3,17 @@
   <ItemGroup>
     <Repository Include="standard" >
       <Organization>dotnet</Organization>
-      <Branch>release/2.0.0</Branch>
+      <Branch>master</Branch>
```

Comment:
Do we even use these properties anywhere? What is the process you use to update these shas?